.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
   "editor.formatOnSave": true,
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
What does this do?

Reply:
i'm guessing this is the new accepted value for this setting. vscode changes this value automatically every time i load this project.